cmis/src/main/java/nl/nn/adapterframework/extensions/cmis/CmisSessionBuilder.java

```diff
@@ -340,17 +339,17 @@ public CmisSessionBuilder setProxyPassword(String string) {
 		return this;
 	}
 
-	public CmisSessionBuilder setUrl(String url) throws ConfigurationException {
+	public CmisSessionBuilder setUrl(String url) {
 		if(StringUtils.isEmpty(url))
-			throw new ConfigurationException("url must be set");
+			throw new IllegalArgumentException("url must be set");
```

Comment:
Op andere plekken in het framework worden dit soort gevallen ConfigurationExceptions in configure(). Kan dat hier niet ook?

Reply:
Het is een builder klasse

Reply:
Klinkt als: dat kan hier niet.